src/main/java/com/ucm/ms/accounts/controllers/UserAccountsController.java

```diff
@@ -66,4 +69,21 @@ public ModelAndView confirmUserAccount(ModelAndView modelAndView, @RequestParam(
             return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
         }
     }
+    
+    
+    
+	@GetMapping("/user_accounts")
```

Comment:
This would mean that the path for this is /api/user_account/user_accounts
We have a method that returns all user accounts (GET for /api/user_account)
I think we should keep this one since it gets the user account based on the user id, but yeah lets change the path to something else.